### PR TITLE
Translations are added in Hindi and Malayalam

### PR DESCRIPTION
**Pull request by gopikaharikumar**
Translations are added in the following languages:
* Hindi
* Malayalam
Closes issue #400

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
 
-    <string name="application_name" translatable="false">Amahi</string>
+    <string name="application_name">Amahi</string>
 
 
     <string name="title_apps">Apps</string>
     <string name="title_settings">Settings</string>
     <string name="title_shares">Shares</string>
-    <string name="title_selected_server">Server</string>
+    <string name="title_selected_server">selected server</string>
```
Comment (cpg) [inline src/main/res/values/strings.xml]: Is there a reason this became lowercase? Seems like a title, so it ought to be "Selected Server" or ... why not just "Server" (less words)?

Comment (cpg) [reply src/main/res/values/strings.xml]: ?

**Comment by gopikaharikumar** [comment]
@cpg can you please review this PR?

**Comment by cpg** [comment]
I made some comments/remarks, but they appear to have been ignored, near as I can tell. That's my review.

**Comment by gopikaharikumar** [comment]
Apologies @cpg , I did not mean to ignore your review. I had made the changes as you told(I have changed the word "selected server" to "Server") and sent the pull request.

**Pull request merged**